app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt

```diff
@@ -232,9 +261,72 @@ class PlayerActivity : AppCompatActivity() {
                 }
                 return true
             }
+
+            override fun onScroll(e1: MotionEvent?, e2: MotionEvent?, distanceX: Float, distanceY: Float): Boolean {
+                if (e1 == null || !isBrightnessVolumeGestureAllowed)
+                    return false
```

Comment:
```suggestion
                e1 as? isBrightnessVolumeGestureAllowed ?: return false
```

Reply:
Uhm isn't this code incorrect? I am new to kotlin so correct me if I am wrong but why should I cast a MotionEvent object to a Boolean object?

![image](https://user-images.githubusercontent.com/20506602/94242060-1f31e500-ff33-11ea-9005-1f84aada73b5.png)

Reply:
Sorry I read the code wrong due too GitHub's syntax highlighting. Ignore this one!